### ISSUE TITLE
Project Status: Does This Project Need Maintenance Help?

### ISSUE THREAD

**Issue by zicklag** [gliheng/flutter-rs#156]
Hey there, I'm new to flutter, but experienced in Rust and I noticed that this project seems to be a little bit inactive since last april or so. I'm likely going to be getting into Flutter development and I wondered if this project could use maintainers to help. There's one PR that's been open for a few months, right now the template repo isn't compiling, and I had to patch `cargo-flutter` to make it compile my projects. Seems like there are some maintenance tasks to take care of.

While I can't guarantee anything about how much time I'll be able to spend on it, I might be able to put some time into helping with this. If Flutter ends up working as well as it looks like it could then this would potentially become an important part of the applications that me and my organization make and I'll have more time to help make sure that this keeps working.

Any thoughts?

**Comment by dlight** [comment]
Perhaps an okay course of action is to fork this repo, apply the PRs, and if the original author ever want to continue development and merge your fork, he can do it at his own pace.

**Comment by dvc94ch** [comment]
I'd really recommend contributing to iced instead. I think it has a more promising future.

@csnewman are you still using flutter-rs? Should we redirect people to the iced project instead?

**Comment by zicklag** [comment]
I just found [nativeshell](https://github.com/nativeshell/nativeshell) about a week ago and it seems to accomplish most of the same goal as flutter-rs, but simpler without requring implementing the windowing and raw flutter integration, I think. I tried it out on Linux and it seems to be working really well.

Seems it's already ready to pretty much supercede this project.

It's pretty exciting.

---

It would probably be good to mark this project as not maintained in the README and probably link to both Iced and NativeShell.

**Comment by dvc94ch** [comment]
Yeah, flutter-rs was written before flutter had a proper backend for desktop.

**Comment by vially** [comment]
I have been working on a [forked version of `flutter-rs`](https://github.com/vially/flust) (with a *much* smaller scope though, only a Wayland backend is currently implemented) and it's starting to get close to a point where it might be useful to other people too.

@dvc94ch, @csnewman: Do you think it makes sense to try to get my changes merged back into *this* repository or should I just continue working on my fork?

Both options would work for me, but if it's the latter I think it would help to mark this repository as archived, just to make it clear that no further development is planned in this repository.

**Comment by vially** [comment]
As a quick update on this, I am planning to start publishing some of my (forked) packages to `crates.io`. Therefore, since I cannot reuse the original crate names, I have renamed my fork to [Flust](https://github.com/vially/flust) and will continue development under that project name going forward.

If the `flutter-rs` project decides to resume development maybe we will find a way to join our efforts (e.g.: by merging the projects back into one, or by sharing crates, etc). If not, I'll just continue development under the Flust project name.
